sdcm/kafka/kafaka_cluster.py

```diff
@@ -0,0 +1,158 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
Nice file name ^_^

Reply:
yep, I keep mistyping it, renamed this file.